production/helm/loki/templates/ingester/statefulset-ingester-zone-c.yaml

```diff
@@ -184,7 +184,7 @@ spec:
           {{- include "loki.configVolume" . | nindent 10 }}
         - name: runtime-config
           configMap:
-            name: {{ template "loki.fullname" . }}-runtime
+            name: {{ template "loki.name" . }}-runtime
```

Comment:
The configmap creation isn't updated as far as I can see. So how will this match that?

Reply:
that's the issue - the configmap name is actually `{{ template "loki.name" . }}-runtime`

Reply:
it looks like the runtime configmap is created with loki.name already?

Reply:
Got it, thanks